views/tool/cardpage.pug

```diff
@@ -2,12 +2,12 @@ extends ../layout
 
 block content
   include ../flash
-  input(type='hidden', id='data', value=JSON.stringify(data))
-  input(type='hidden', id='card', value=JSON.stringify(card))
-  input(type='hidden', id='prices', value=JSON.stringify(prices))
-  input(type='hidden', id='related', value=JSON.stringify(related))
-  input(type='hidden', id='cubes', value=JSON.stringify(cubes))
+
   #react-root
+
+block scripts
   include ../react
+  script(type='text/javascript').
+    window.reactProps = !{reactProps};
```

Comment:
I copied this from other pages, but I'm not sure what this syntax means. If someone could explain it to me, I'd love to learn :)